ketcher-autotests/tests/Macromolecule-editor/Polymer-Bond-Tool/connection-rules-for-chems-monomers.spec.ts

```diff
@@ -1252,39 +1228,6 @@ test.describe('Connection rules for chems: ', () => {
     );
   }
 
-  Object.values(chemMonomers).forEach((leftMonomer) => {
-    Object.values(molecules).forEach((rightMolecule) => {
-      /*
-       *  Test task: https://github.com/epam/ketcher/issues/5960
-       *  Description: Verify that connection points between monomers and molecules can be created by drawing bonds in macro mode
-       *  Case: Monomer center to molecule atom connection
-       *  Step: 1. Load monomer (chem) and shift it to the left
-       *        2. Load molecule (system loads it at the center)
-       *        3. Drag center of monomer to first (0th) atom of molecule
-       *        Expected result: No connection should be establiched
-       *  WARNING: That test tesults are wrong because of bug: https://github.com/epam/ketcher/issues/5976
-       *  Screenshots must be updated after fix and fixme should be removed
-       */
-      test(`Case 11: Connect Center of Chem(${leftMonomer.alias}) to atom of MicroMolecule(${rightMolecule.alias})`, async () => {
```

Comment:
If we deleted tests, then please delete screenshots related to it

Reply:
Deleted